File: api/code-samples/python/klay/transaction/getTransactionByBlockNumberAndIndex.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x27"
transactionIndex = "0x0"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_transaction_by_block_number_and_index(blockTag, transactionIndex)

print(json.loads(klay_response.response.data))

File: api/code-samples/javascript/klay/transaction/getTransactionByBlockNumberAndIndex.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new Caver.ApiClient("https://api.baobab.klaytn.net:8651"));


    const blockNumber = 118593751
    const transactionIndexPosition = '0x0'

    sdk.klay.getTransactionByBlockNumberAndIndex(blockNumber, transactionIndexPosition, {}, (err, data, response) => {
        console.log(data);
    });

}
)()